frontend/src/app/interfaces/electrs.interface.ts

```diff
@@ -76,6 +76,7 @@ export interface Vin {
   issuance?: Issuance;
   // Custom
   lazy?: boolean;
+  vinId?: string; // `txid:index` where txid links to the transaction this input is spent in, and index is the position of this input within this transaction
```

Comment:
I'd prefer not to add extra optional fields to these backend interfaces where possible.

If we'll be using it in lots of places I think it's better to add a new interface extending this one where the field is required (then you get the type checking benefits of knowing it's always available in that context).